README.pt.md

```diff
@@ -11,7 +11,7 @@ O SDK oficial do Mercado Pago.
 
 ## 💡 Requisitos
 
-**.NET Standard 2.0+**, **.NET Core 2.0+**, e **.NET Framework 4.6.1+**.
+**.NET Standard 2.1+**, **.NET Core 2.0+**, y **.NET Framework 6.0+**.
```

Comment:
```suggestion
**.NET Standard 2.1+**, **.NET Core 2.0+**, e **.NET Framework 6.0+**.
```

Translate word SOPORTE to SUPORTE in the final file
<img width="254" alt="image" src="https://github.com/user-attachments/assets/1069a2a2-084f-4049-b789-0a54f7a35b90">